### DISCUSSION TITLE
Optional dependency not satisfied with wurlitzer

### DISCUSSION THREAD

**Discussion by qwuaren** [mapequation/infomap#180] [Q&A] [answered]
Hi. I'm trying to install infomap on a windows computer to run this example:

    from cdlib import algorithms
    import networkx as nx
    G = nx.karate_club_graph()
    coms = algorithms.infomap(G)

However after installing infomap by "pip install infomap" I get the error message:

    ModuleNotFoundError: Optional dependency not satisfied: install package wurlitzer to use infomap.

This error message persists after installing the wurlitzer package. I can't figure out what I need to do to fix it. I am running python via anaconda, v3.8, and have reinstalled the Windows c++ build tools. I'm a bit inexperienced with using Python for this.

**Comment by antoneri** [comment, marked as answer]
Hi!

Try 

```python
from infomap import Infomap
```

instead of CDlib. 

You can find an example in [infomap-networkx.py](https://github.com/mapequation/infomap/blob/master/examples/python/infomap-networkx.py). 

Hope this helps!

**Reply by qwuaren** [reply]
Thank you, I managed to make that work instead! =)
